translations/ja.json

```
@@ -500,56 +500,56 @@
-	"Running remotely (CPUs={{.number_of_cpus}}, Memory={{.memory_size}}MB, Disk={{.disk_size}}MB) ...": "",
-	"SSH key (ssh driver only)": "",
-	"SSH port (ssh driver only)": "",
-	"SSH user (ssh driver only)": "",
-	"Save a image from minikube": "",
-	"Select a valid value for --dnsdomain": "",
-	"Send trace events. Options include: [gcp]": "",
-	"Service '{{.service}}' was not found in '{{.namespace}}' namespace.\nYou may select another namespace by using 'minikube service {{.service}} -n \u003cnamespace\u003e'. Or list out all the services using 'minikube service list'": "",
+	"Returns logs to debug a local Kubernetes cluster": "ローカルの Kubernetes クラスターをデバッグするためのログを返します",
+	"Returns the Kubernetes URL for a service in your local cluster. In the case of multiple URLs they will be printed one at a time.": "ローカルクラスター中のサービス用 Kubernetes URL を返します。複数 URL の場合、それらは一度に出力されます。",
+	"Returns the value of PROPERTY_NAME from the minikube config file.  Can be overwritten at runtime by flags or environmental variables.": "minikube 設定ファイル中の PROPERTY_NAME の値を返します。実行時にフラグか環境変数を用いて上書きできます。",
+	"Right-click the PowerShell icon and select Run as Administrator to open PowerShell in elevated mode.": "PowerShell を特権モードで開くために、PowerShell アイコンを右クリックし、管理者として実行を選択してください。",
+	"Run 'kubectl describe pod coredns -n kube-system' and check for a firewall or DNS conflict": "'kubectl describe pod coredns -n kube-system' を実行し、ファイアウォールか DNS 衝突を確認してください",
+	"Run 'minikube delete' to delete the stale VM, or and ensure that minikube is running as the same user you are issuing this command with": "ストールした VM を削除するため、'minikube delete' を実行するか、次のコマンドを実行した時と同じユーザーとして minikube を実行している事を確認してください",
+	"Run 'sudo sysctl fs.protected_regular=0', or try a driver which does not require root, such as '--driver=docker'": "'sudo sysctl fs.protected_regular=0' を実行するか、'--driver=docker' のような root を必要としないドライバーを試してください",
+	"Run a kubectl binary matching the cluster version": "クラスターのバージョンに一致する kubectl バイナリーを実行します",
+	"Run minikube from the C: drive.": "C: ドライブから minikube を実行してください。",
+	"Run the Kubernetes client, download it if necessary. Remember -- after kubectl!\n\nThis will run the Kubernetes client (kubectl) with the same version as the cluster\n\nNormally it will download a binary matching the host operating system and architecture,\nbut optionally you can also run it directly on the control plane over the ssh connection.\nThis can be useful if you cannot run kubectl locally for some reason, like unsupported\nhost. Please be aware that when using --ssh all paths will apply to the remote machine.": "Kubernetes クライアントを実行します (必要であればクライアントをダウンロードします)。kubectl の後に -- を忘れないでください!\n\nこれは、クラスターと同じバージョンの Kubernetes クライアント (kubectl) を実行します\n\n通常、ホスト OS とアーキテクチャに一致するバイナリーをダウンロードしますが、\nそのほかに SSH 接続経由でコントロールプレーン上で kubectl を直接実行することもできます。\nこれは、いくつかの理由によりローカルで kubectl を実行できない場合に便利です。\n--ssh を使用する場合、全パスがリモートマシンに適用されることに注意してください。",
```

This is very long message.
There is not translationed string of `like unsupported host` .

```suggestion
	"Run the Kubernetes client, download it if necessary. Remember -- after kubectl!\n\nThis will run the Kubernetes client (kubectl) with the same version as the cluster\n\nNormally it will download a binary matching the host operating system and architecture,\nbut optionally you can also run it directly on the control plane over the ssh connection.\nThis can be useful if you cannot run kubectl locally for some reason, like unsupported\nhost. Please be aware that when using --ssh all paths will apply to the remote machine.": "Kubernetes クライアントを実行します (必要であればクライアントをダウンロードします)。kubectl の後に -- を忘れないでください!\n\nこれは、クラスターと同じバージョンの Kubernetes クライアント (kubectl) を実行します\n\n通常、ホスト OS とアーキテクチャに一致するバイナリーをダウンロードしますが、\nそのほかに SSH 接続経由でコントロールプレーン上で kubectl を直接実行することもできます。\nこれは、未サポートホストなど、いくつかの理由によりローカルで kubectl を実行できない場合に便利です。\n--ssh を使用する場合、全パスがリモートマシンに適用されることに注意してください。",
```